samples/exceptions/ExceptionSamples.java

```diff
@@ -0,0 +1,35 @@
+package exceptions;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ExceptionSamples {
+    public static void run() {
+
+        int a = 15;
+        int b = 0;
+
+        try {
+            System.out.println(divide(10, 0));
```

Comment:
Also it is a bit better to first run `divide()` method, then print the result inside `System.out.println()`.